pkg/operator/ceph/config/livenessprobe_test.go

```diff
@@ -76,51 +76,70 @@ func configLivenessProbeHelper(t *testing.T, keyType rookv1.KeyType) {
 }
 
 func TestGetLivenessProbeWithDefaults(t *testing.T) {
-	defaultProbe := &v1.Probe{
-		Handler: v1.Handler{
-			Exec: &v1.ExecAction{
-				// Example:
-				Command: []string{
-					"env",
-					"-i",
-					"sh",
-					"-c",
-					"ceph --admin-daemon /run/ceph/ceph-mon.c.asok mon_status",
+	t.Run("using default probe", func(t *testing.T) {
+		defaultProbe := &v1.Probe{
+			Handler: v1.Handler{
+				Exec: &v1.ExecAction{
+					// Example:
+					Command: []string{
+						"env",
+						"-i",
+						"sh",
+						"-c",
+						"ceph --admin-daemon /run/ceph/ceph-mon.c.asok mon_status",
+					},
 				},
 			},
-		},
-		InitialDelaySeconds: 10,
-	}
-	// in case of default probe
-	desiredProbe := &v1.Probe{}
-	desiredProbe = GetLivenessProbeWithDefaults(desiredProbe, defaultProbe)
-	assert.Equal(t, desiredProbe, defaultProbe)
+			InitialDelaySeconds: 10,
+		}
+		// in case of default probe
+		desiredProbe := &v1.Probe{}
+		desiredProbe = GetLivenessProbeWithDefaults(desiredProbe, defaultProbe)
+		assert.Equal(t, desiredProbe, defaultProbe)
+	})
 
-	// in case of overriding probe
-	desiredProbe = &v1.Probe{
-		Handler: v1.Handler{
-			Exec: &v1.ExecAction{
-				// Example:
-				Command: []string{
-					"env",
-					"-i",
-					"sh",
-					"-c",
-					"ceph --admin-daemon /run/ceph/ceph-osd.0.asok status",
+	t.Run("overriding default probes", func(t *testing.T) {
+		defaultProbe := &v1.Probe{
+			Handler: v1.Handler{
+				Exec: &v1.ExecAction{
+					// Example:
+					Command: []string{
+						"env",
+						"-i",
+						"sh",
+						"-c",
+						"ceph --admin-daemon /run/ceph/ceph-mon.c.asok mon_status",
```

Comment:
```suggestion
						"ceph --admin-daemon /run/ceph/ceph-mon.b.asok mon_status",
```
let's change it to different command so that we can test below(line 138) the overriding probe is not happening. currently, we have same in both place desired and default prbe.